testsuite/gnat2goto/tests/string_literal/string_literal.adb

```diff
@@ -1,5 +1,8 @@
 procedure string_literal is
    A : constant String := "Hello";
+   B : constant String := "Hello";
+   C : constant String := "Goodbye";
 begin
-   null;
+   pragma Assert(A = B);
+   pragma Assert(A /= C);
```

Comment:
This commit seems unrelated.  I'm OK with it I just wonder if it would be happier in a different PR.

Reply:
It could be in a new PR (happy to do that if you'd prefer) - but I had to tweak string literals a bit to make the submodule bump still pass this regression test, and in doing so I was able to make this test actually do some thing semi-useful (still more work to do to have string literals working properly though).